src/plugins/workspace/server/workspace_client.ts

```diff
@@ -95,7 +95,7 @@ export class WorkspaceClient implements IWorkspaceClientImpl {
       const existingWorkspaceRes = await this.getScopedClientWithoutPermission(requestDetail)?.find(
         {
           type: WORKSPACE_TYPE,
-          search: attributes.name,
+          search: `"${attributes.name}"`,
```

Comment:
does this search for a workspace including this exact phrase or one that is an exact match?

Reply:
It depends on the field type. For my case, since `workspace.name` filed type is keyword, it will convert to `TermQuery` that's exact match.